docs/source/getting_started/amd-installation.rst

```diff
@@ -0,0 +1,80 @@
+.. _installation:
```

Comment:
Might worth changing the main README.md to summarize the work you have done to support vllm on ROCM, and direct readers to this file.

Reply:
Thank you for the suggestion. We are working on this.